Detectors/EMCAL/calibration/testWorkflow/EMCALChannelCalibratorSpec.h

```diff
@@ -160,6 +174,28 @@ class EMCALChannelCalibDevice : public o2::framework::Task
         }
       }
     }
+    static bool firstCall = true;
+    if (firstCall) {
+      firstCall = false;
+      if (mTimeCalibrator) {
+        mTimeCalibrator->loadSavedSlot();
+      } else if (mBadChannelCalibrator) {
+        mBadChannelCalibrator->loadSavedSlot();
+      }
+    }
+
```

Comment:
Could you make this compatible with ECS stop/start procedure (i.e. starting a new run within the same O2 session, see https://indico.cern.ch/event/1270050/contributions/5335478/attachments/2620040/4530862/EOS_and_SSS.pdf)